src/dmd/hdrgen.d

```diff
@@ -390,9 +390,8 @@ public:
         buf.writestring("if (");
         if (Parameter p = s.prm)
         {
-            StorageClass stc = p.storageClass;
-            if (!p.type && !stc)
-                stc = STC.auto_;
+            const stc = (p.type || p.storageClass) ? p.storageClass : STC.auto_;
```

Comment:
This is still unrelated, no?

Reply:
It's a workaround for https://issues.dlang.org/show_bug.cgi?id=20466